### PR TITLE
Fix doc errors in concepts and getting started

### PR DESCRIPTION
**Pull request by sp1999**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/getting-started/vsphere/overview.md [hunk 1]
```diff
@@ -39,10 +39,10 @@ To begin, the cluster creation process runs a series of [govc](https://github.co
...
 * Using the URL and credentials provided in the cluster spec files, authenticates to the vSphere provider.
 
-* Validates the datacenter and the datacenter network exists:
+* Validates that the datacenter and the datacenter network exists.
 
 * Validates that the identified datastore (to store your EKS Anywhere cluster) exists, that the folder holding your EKS Anywhere cluster VMs exists, and that the resource pools containing compute resources exist.
-If you have multiple `VSphereMachineConfig` objects in your config file, will see these validations repeated:
+If you have multiple `VSphereMachineConfig` objects in your config file, you will see these validations repeated.
 
 * Validates the virtual machine templates to be used for the control plane and worker nodes (such as `ubuntu-2004-kube-v1.20.7`):
```
Comment (ahreehong) [inline docs/content/en/docs/getting-started/vsphere/overview.md:47]:
```suggestion
* Validates the virtual machine templates to be used for the control plane and worker nodes (such as `ubuntu-2004-kube-v1.20.7`).
```

Comment (sp1999) [reply docs/content/en/docs/getting-started/vsphere/overview.md]: Updated

**Pull request merged**